### ISSUE TITLE
[Suggestion] Shipping a CDN build with all variants

### ISSUE THREAD

**Issue by adevade** [tailwindlabs/tailwindcss#1318]
This might be an insane suggestion, but I thought I'd throw it out there anyway.

It would be nice to have a compiled file with all variants for when hacking things together on CodePen or other online editors.

I'm afraid I might underestimate the file size though...?

Thanks for all your work, and have a nice day!

**Comment by Apokly** [comment]
Hi,

Are you looking for CDN URL for every releases ? Might I suggest you https://cdn.jsdelivr.net/npm/tailwindcss@1.2.0-canary.5/dist/tailwind.min.css, for latest current release ?

Hope this helps.

**Comment by adevade** [comment]
Hey @Apokly! Thanks for replying! 

I know Tailwind is available via CDN. However, I was using it on CodePen and needed the `disabled:` variant, which isn't in the default config. What I want is a build with all variants enabled.

When I tried enabling all variants ([as in the docs](https://tailwindcss.com/docs/configuring-variants/#enabling-all-variants)) the unminified, uncompressed build came out at about **7 MB**. Yeesh..

I then minified it (using [clean-css-cli](https://www.npmjs.com/package/clean-css-cli)) I got it down to **5.6 MB**. Not a very large difference.

With gzip compression though, it shrunk down to only **534 KB(!!)**. I think this would be viable to have on a CDN, right?
